wis2box-management/Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN apt-get update -y && apt-get install -y ${DEBIAN_PACKAGES} \
     https://github.com/wmo-cop/pyoscar/archive/refs/tags/0.7.0.zip \
     https://github.com/geopython/pygeometa/archive/refs/tags/0.16.0.zip \
     https://github.com/wmo-im/pywis-topics/archive/refs/tags/0.2.0.zip \
+    # install cap validator
+    && pip3 install --no-cache-dir capvalidator>=0.1.0-dev2 \
```

Comment:
Can we make the capvalidator pin consistent between here and `wis2box-management/requirements.txt`?